app/package.json

```diff
@@ -27,9 +27,9 @@
   "author": "Carlo Maria Curinga",
   "license": "Apache-2.0",
   "bugs": {
-    "url": "https://github.com/resin-io/resin-electronjs/issues"
+    "url": "https://github.com/balena-io/balena-electronjs/issues"
```

Comment:
This URL doesn't work yet - I guess somebody with the necessary privileges will need to rename this repo after this PR is merged?

Reply:
Yeah, we could ask @shaunmulligan to do it I believe

Reply:
@dansku yeah can update it once merged, if you give me a ping.

Reply:
@shaunmulligan will do!

Reply:
ping @shaunmulligan I guess it's time for this repo to be renamed? (as this PR has been merged)